modules/serverless-jaeger-config.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * /serverless/serverless-tracing.adoc
+// */jaeger/jaeger_config/serverless-jaeger-integration.adoc
+
+[id="serverless-jaeger-config_{context}"]
+= Configuring Jaeger for use with {ServerlessProductName}
+
+.Prerequisites
+
+To configure Jaeger for use with {ServerlessProductName}, you will need:
+
+* Cluster administrator permissions on an {product-title} cluster.
+* A current installation of {ServerlessOperatorName} and Knative Serving.
+* A current installation of the Jaeger Operator.
+
+.Procedure
+
+. Create and apply the Jaeger custom resource:
++
+----
+$ cat <<EOF | oc apply -f -
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: jaeger
+EOF
+----
+
+. Enable tracing for Knative Serving, by editing the `KnativeServing` resource and adding a YAML configuration for tracing.
++
+.Tracing YAML example
+----
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  config:
+    tracing:
+      sample-rate: "0.1"
+      backend: zipkin <1>
+      zipkin-endpoint: http://jaeger-collector.<JAEGER_NAMESPACE>.svc.cluster.local:9411/api/v2/spans <2>
+      debug: "true"
```

Comment:
I think we should not set the debug to true here.  (Certainly not without an explanation of what it does?)

Reply:
@markusthoemmes @nak3 what do you think?

Reply:
Yes, agree. We should not set to false.

```suggestion
      debug: "false"
```

(If we add an explanation, we can note as "Enabling debug mode allows all spans to be sent to the server bypassing sampling.", which is just same explanation in [upstream](https://github.com/knative/serving/blob/a0c33cf6d3c8645118ded05b7399d428ec970650/config/core/configmaps/tracing.yaml#L52-L53) )